contrib/utilities/indent

```diff
@@ -31,22 +45,26 @@ if test ! -f contrib/utilities/astyle.rc ; then
   exit 1
 fi
 
-if test -z "`which astyle`" ; then
+astyle="$(find_astyle)"
+
+if test -z "${astyle}"; then
   echo "*** No astyle program found."
   exit 1
 fi
 
-if test "`astyle --version 2>&1`" != "Artistic Style Version 2.04" ; then
+if test "$("${astyle}" --version 2>&1)" != "Artistic Style Version 2.04" ; then
   echo "*** Found a version of astyle different than the required version 2.04."
   exit 1
 fi
 
+echo "--- Found astyle 2.04 at ${astyle}"
+
 
 # collect all header and source files and process them in batches of 50 files
 # with up to 10 in parallel
 echo "--- Indenting all ASPECT header and source files"
 
-find tests include source benchmarks cookbooks unit_tests \( -name '*.cc' -o -name '*.h' -o -name '*.h.in' \) -print | xargs -n 50 -P 10 astyle --options=contrib/utilities/astyle.rc
+find tests include source benchmarks cookbooks unit_tests \( -name '*.cc' -o -name '*.h' -o -name '*.h.in' \) -print | xargs -n 50 -P 10 "${astyle}" --options=contrib/utilities/astyle.rc
```

Comment:
do you need the quotes here?

Reply:
Without the quotes this would fail if there are spaces in the executable's path.